cmd/createorupdate/update.go

```diff
@@ -43,6 +106,49 @@ func update(ctx context.Context, cs *api.OpenShiftManagedCluster, p api.Plugin)
 	return nil
 }
 
+type scaleOp struct {
+	up   bool
+	down bool
+	// diff is the count of VMs that are going
+	// to be scaled up or down.
+	diff int
```

Comment:
pretty sure you can't do this - it's not idempotent

Reply:
I can get rid of this but in order to make scale indempotent we always need to check for the health of all VMs, right?

Reply:
Can you walk me through a scenario where this is not idempotent?

Reply:
I don't think you need to store the delta here in order to be able solely to health check newly created VMs.  You do need to list the VMs pre- and post- deploy though but that code is elsewhere.

Reply:
FWIW, this is only used by scale down where we don't need to list post-deploy (deploy is actually a no-op). I went ahead and updated it to the desired count of the scale set.